File: library/src/main/java/com/getstream/sdk/chat/rest/response/MessageResponse.java
package com.getstream.sdk.chat.rest.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.getstream.chat.android.client.models.Message;

public class MessageResponse {
    @SerializedName("message")
    @Expose
    private Message message;

    public Message getMessage() {
        return message;
    }

    public void setMessage(Message message) {
        this.message = message;
    }
}

File: stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/response/MessageResponse.kt
package io.getstream.chat.android.client.api2.model.response

import com.squareup.moshi.JsonClass
import io.getstream.chat.android.client.api2.model.dto.DownstreamMessageDto

@JsonClass(generateAdapter = true)
internal data class MessageResponse(
    val message: DownstreamMessageDto,
)
